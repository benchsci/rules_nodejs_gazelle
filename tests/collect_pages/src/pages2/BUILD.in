# gazelle:js_nextjs_pages_root
# gazelle:js_nextjs_pages_root_name next_pages2
