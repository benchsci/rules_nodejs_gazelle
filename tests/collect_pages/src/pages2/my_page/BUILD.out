load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

ts_project(
    name = "my_page",
    srcs = [
        "index.ts",
        "module_file.ts",
        "subpage/index.ts",
    ],
)
